spec/features/bulk_resource_delete_spec.rb

```diff
@@ -0,0 +1,126 @@
+      'traject_context_source' => 'matching_resource' }
+  end
+
+  before do
+    perform_enqueued_jobs do
+      resource_1.save_and_index
+      resource_2.save_and_index
+      resource_3.save_and_index
+      resource_4.save_and_index
+    end
+  end
+
+  after do
+    allow(ResourceRemover).to receive(:remove_resource).and_call_original # undo any funny business from the individual tests
+
+    # RSpec will clean up RDBMS entries automatically, but we also want to clean up solr entries after this test
+    [resource_1, resource_2, resource_3, resource_4].each do |resource|
+      ResourceRemover.remove_resource(resource: resource)
+    end
+  end
+
+  context 'with a curator' do
+    before do
+      login_as curator
+    end
+
+    it 'deletes exactly the resources in the search results' do # rubocop:disable RSpec/ExampleLength
+      visit root_path
+
+      fill_in 'Search...', with: 'matching_resource'
+      click_button 'Search'
+      within '#documents' do
+        expect(page).to have_css '.document', count: 2
+      end
+
+      click_button 'Bulk actions'
+      click_link 'Delete items'
+      perform_enqueued_jobs do
+        click_button 'Delete'
+      end
+
+      job_tracker = Spotlight::JobTracker.recent.take # grab the most recent job tracker from the DB
+      expect(job_tracker).to be_completed
+
+      fill_in 'Search...', with: 'matching_resource'
+      click_button 'Search'
+      within '#documents' do
+        expect(page).to have_css '.document', count: 0
+      end
+
+      fill_in 'Search...', with: 'not_in_the_original_search_results'
+      click_button 'Search'
+      within '#documents' do
+        expect(page).to have_css '.document', count: 2
+      end
+    end
+
+    context 'when an error is thrown trying to delete a resource' do
+      before do
+        allow(ResourceRemover).to receive(:remove_resource).and_raise(RSolr::Error::ConnectionRefused, 'network blip')
+      end
+
+      it 'gracefully handles the unexpected error and logs the event appropriately' do # rubocop:disable RSpec/ExampleLength
+        visit root_path
+
+        fill_in 'Search...', with: 'matching_resource'
+        click_button 'Search'
+        within '#documents' do
+          expect(page).to have_css '.document', count: 2
+        end
+
+        click_button 'Bulk actions'
+        click_link 'Delete items'
+        perform_enqueued_jobs do
+          click_button 'Delete'
+        end
+
+        job_tracker = Spotlight::JobTracker.recent.take # grab the most recent job tracker from the DB
+        puts "JM_LOG job_tracker.status=#{job_tracker.status}"
+        expect(job_tracker).to be_failed
+      end
+    end
+  end
+
+  context 'with an ordinary user' do
+    it 'does not offer bulk delete' do
+      visit root_path
+      click_button 'Search'
+      expect(page).not_to have_content 'Bulk actions'
+      expect(page).not_to have_content 'Delete items'
+    end
```

Comment:
i don't know if it's worth the time, since the behavior is inherited from spotlight, and seems to work as expected here... but i had also considered adding a request spec that shows that authorization checks are enforced when hitting the new controller method for bulk delete (so as to prevent savvy but maliscious users from constructing a delete request despite the lack of button in the UI).